stable/openstack-base/bundle.yaml

```diff
@@ -329,15 +342,17 @@ applications:
     annotations:
       gui-x: '300'
       gui-y: '1550'
-    charm: cs:rabbitmq-server-117
+    charm: ch:rabbitmq-server
+    channel: 3.8/stable
```

Comment:
we test stable/yoga branches with the 3.9/edge track, hence we should use `3.9/stable` here

Reply:
ack